scripts/tuple_tree_generator/tuple_tree_generator/templates/all_types_variant_decls.h.tpl

```diff
@@ -0,0 +1,25 @@
+# pragma once
+
+#include <variant>
+
+/*#-
+This template file is distributed under the MIT License. See LICENSE.md for details.
+The notice below applies to the generated files.
+#*/
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+// This file is autogenerated! Do not edit it directly
+
+
+// Forward declarations
+namespace /*= namespace =*/ {
+
+using AllTypes = std::variant<
+/** for name in all_types|sort **/
+/*= name =*//** if not loop.last **/,/** endif **/
```

Comment:
Gosh, where is `loop` defined? Some jinja magic?

Reply:
no idea, i did as cremo told me to, i guess so.

Reply:
You did not answer `:P`.
Ask cremo and answer.